crates/bevy_render/src/entity.rs

```diff
@@ -77,3 +86,40 @@ impl Default for Camera2dBundle {
         }
     }
 }
+
+/// A component bundle for "orthographic camera" entities
+///
+/// Uses a normalized orthographic projection.
+///
+/// Use this for CAD-like 3D views or non-pixel-oriented 2D games.
+#[derive(Bundle)]
+pub struct CameraOrthoBundle {
+    pub camera: Camera,
+    pub orthographic_projection: ScaledOrthographicProjection,
+    pub visible_entities: VisibleEntities,
+    pub transform: Transform,
+    pub global_transform: GlobalTransform,
+}
+
+impl Default for CameraOrthoBundle {
+    fn default() -> Self {
+        // FIXME does this same far trick from `Camera2DBundle` make any sense here?
```

Comment:
My guess is that the correct answer here is no, it doesn't make sense.

I think a sane default would be located at `(100, 100, 100)`, looking at `(0,0,0)`, and scaled to `50x`, i.e. include more of the scene.

But I think that something similar would also apply to perspective transforms.

Reply:
Yes, I thought so too. Whatever the 3D perspective camera does is probably right for this too.

Reply:
For now, I will just make it initialize to default (like the perspective projection). That seems like it would be less surprising to users than hardcoding some arbitrary values.